testpar/t_span_tree.c

```diff
@@ -35,6 +35,8 @@
 #include "H5private.h"
 #include "testphdf5.h"
 
+#define LOWER_DIM_SIZE_COMP_TEST__RUN_TEST__DEBUG 0
```

Comment:
Moving this up in the file eliminates a warning about it being undefined and evaluating to 0